.travis.yml

```diff
@@ -22,7 +22,7 @@ matrix:
       python: 2.7
     - env: TASK="ci-checks ci-packs-tests"
       python: 2.7
-    - env: TASK="compilepy3 ci-st2client-py3-tests" CACHE_NAME=py3
+    - env: TASK="compilepy3 ci-py3-unit-tests" CACHE_NAME=py3
```

Comment:
NIT: Even though, `ci-py3-unit-tests` could be `ci-unit-py3`, following the existing `ci-unit` task we have in the build matrix